sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/unresolved.scala

```diff
@@ -141,11 +144,11 @@ case class UnresolvedFunction(
   override def nullable: Boolean = throw new UnresolvedException(this, "nullable")
   override lazy val resolved = false
 
-  override def prettyString: String = {
-    s"${name}(${children.map(_.prettyString).mkString(",")})"
+  override def sql: String = {
+    s"$name(${children.map(_.sql).mkString(", ")})"
```

Comment:
Is this same of the default one in `Expression`? after we override `prettyName` with `name`